components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -821,7 +837,7 @@ void RewardsServiceImpl::OnWalletInitialized(ledger::Result result) {
     ready_->Signal();
   }
 
-  if (result == ledger::Result::WALLET_CREATED) {
+  if (is_wallet_initialized_) {
```

Comment:
After #5647 (landed in 1.11), we are no longer calling `StartNotificationTimers` on ledger/browser startup because `result` from `bat_ledger_->Initialize` is `ledger::Result::LEDGER_OK`.